### ISSUE TITLE
API

### ISSUE THREAD

**Issue by xHyroM** [BentoBoxWorld/Bank#33]
### Is your feature request related to a problem?

Is there any API like in Levels? Placeholders are good but slow.

### Describe the solution you'd like.

Javadocs, API

### Describe alternatives you've considered.

using placeholders but slow ._.

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest supported BentoBox build or Addons.

**Comment by tastybento** [comment]
Yep, here you go: https://ci.codemc.io/job/BentoBoxWorld/job/Bank/ws/target/apidocs/index.html

I expect what you'll want to access is the BankManager object (you can get it by calling a method in Bank), and then here are the various methods:
https://ci.codemc.io/job/BentoBoxWorld/job/Bank/ws/target/apidocs/world/bentobox/bank/BankManager.html

I assume you are trying to access this from another plugin? Let me know if it works for you.
